build.gradle

```diff
@@ -690,8 +690,6 @@ project(':streams') {
     // this dependency should be removed after KIP-4
     compile (libs.zkclient) {
       // users should be able to choose the logging implementation (and slf4j bridge)
-      exclude module: 'slf4j-log4j12'
```

Comment:
Why did you remove this line? I thought we only needed to remove the subsequent line and to add slf4j-log4j12 in streams-examples.